### ISSUE TITLE
Auth.JS error.

### ISSUE THREAD

**Issue by jhritz89** [MMRIZE/MMM-DropboxPictures#2]
When attempting to run "node auth.js" I got a PKCE warning when accessing the URL.

Error: You must use PKCE when generating the authorization URL to not include a client secret
    at DropboxAuth.getAccessTokenFromCode (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/dropbox/cjs/src/auth.js:379:17)
    at /home/pi/MagicMirror/modules/MMM-DropboxPictures/auth.js:33:12
    at Layer.handle [as handle_request] (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/layer.js:95:5)
    at /home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/index.js:346:12)
    at next (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/router/index.js:280:10)
    at expressInit (/home/pi/MagicMirror/modules/MMM-DropboxPictures/node_modules/express/lib/middleware/init.js:40:5)

Running node.js 22.2.0

**Comment by eouia** [comment]
Hmmm.. I made some mistake on README about installation. Could you try reinstall?

**Comment by jhritz89** [comment]
I got it! I was trying to run this via SSH then accessing it via IP/hostname from another workstation, instead of localhost. Logged in via VNC then ran from terminal, and it worked.

User error.

Also, I copied the template in the readme and modified as needed. I had to remove the ... near the end as it showed a syntax error.

{
  module: 'MMM-DropboxPictures',
  position: 'top_left', // `fullscreen_below` is more usual.
  //header: "DROPBOX",
  config: {
    verbose: true,
    imageLife: 1000 * 60 * 30, // 30 minutes
    directory: "/Photos",
    fileNames: [ "DSC", /^IMG/ ],
    width: "400px", // For fullscreen, "100%" would be fit.
    height: "400px",
    ...
  }
},

After this, I'm back up and running.
